src/serial/remote/mod.rs

```diff
@@ -25,6 +25,22 @@ mod protocol_v3;
 mod protocol_v4;
 pub mod riscv_debug;
 
+pub struct RemoteCommands;
+
+#[allow(unused)]
+impl RemoteCommands
```

Comment:
NB that the lifetimes of all the strings here should be implicitly static (so you should be able to say just `&str`) - if that is not the case then the typing is fine, but we should leave the lifetime implicit if possible.